app/services/gmail/labels.py

```
@@ -241,29 +245,30 @@ def _apply_label_operation_background(
-            state.label_operation_status["progress"] = 40 + int(
-                (affected / total_emails) * 60
-            )
-            state.label_operation_status["message"] = progress_message_template.format(
-                count=affected, total=total_emails
+            progress = 40 + int((affected / total_emails) * 60)
+            state.update_label_operation_status(
+                affected_count=affected,
+                progress=progress,
+                message=progress_message_template.format(count=affected, total=total_emails),
             )
     except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)